modules/osd-intro.adoc

```diff
@@ -42,17 +42,8 @@ Other enhancements to Kubernetes in {product-title} include improvements in soft
 [id="telemetry_{context}"]
 == Internet and Telemetry access for {product-title}
 
-In {product-title}, you require access to the Internet to install your cluster. The Telemetry service, which runs by default to provide metrics about cluster health and the success of updates, also requires Internet access. If your cluster is connected to the Internet, Telemetry runs automatically, and your cluster is registered to the {console-redhat-com}.
+In {product-title}, you require access to the internet to install and upgrade your cluster.
 
-Once you confirm that your Red Hat OpenShift Cluster Manager inventory is correct, either maintained automatically by Telemetry or manually using OCM, use link:https://access.redhat.com/documentation/en-us/subscription_central/2020-04/html/getting_started_with_subscription_watch/con-how-to-select-datacollection-tool_assembly-requirements-and-your-responsibilities-ctxt#red_hat_openshift[subscription watch] to track your {product-title} subscriptions at the account or multi-cluster level.
+Telemetry is the term that Red Hat uses to describe the information being sent to Red Hat by the OpenShift Telemeter Client. Lightweight attributes are sent from {product-title} clusters to Red Hat to enable subscription management automation, monitor the health of clusters, assist with support, and improve customer experience.
```

Comment:
nit but I wouldn't mention the Telemeter client which is an implementation detail and subject to change.

Reply:
@simonpasquier thank you for your review and for this suggestion. I will remove the reference to the Telemeter Client from this section.